test/libwhisper/shhrpc.cpp

```diff
@@ -99,6 +100,8 @@ struct Setup
 	{
 		dev::p2p::NodeIPEndpoint::test_allowLocal = false;
 	}
+
+	TestOutputHelper testHelper;
```

Comment:
I think this is wrong - what is the purpose of the `passTest` function and others? When will it be called? Will it be called if the testHelper is just a member instead of the fixture inheriting from it?